docs-src/go/chapter-durable-execution/non-deterministic-code-changes.md

```diff
@@ -0,0 +1,70 @@
+---
+id: non-deterministic-code-changes
+title: Non-deterministic code changes
+description: History Replay, sometimes also called Workflow Replay, is the mechanism that Temporal uses to reconstruct the state of a Workflow Execution. Temporal provides Durable Execution via this Replay Functionality.
+sidebar_label: Durability through Replays
+tags:
+  - go sdk
+  - developer-guide-doc-type
+  - event history
+  - replay
+  - durable execution
+---
+
+The most important thing to take away from the section is to make sure you have an application versioning plan whenever you are developing and maintaining a Temporal Application that will eventually deploy to a production environment.
+
+We cover versioning APIs and versioning strategies in other parts of the developer's guide, this chapter set the stage to understand why and how to approach those strategies.
```

Comment:
https://developers.google.com/style/person?hl=en

Did we start using `we` in the docs? I don't remember if we have exceptions to the rule for `We cover` like we do `We recommened` ?